### DISCUSSION TITLE
Auto-summaries of markdown files are not indented anymore?

### DISCUSSION THREAD

**Discussion by aurelien-reeves** [community/community#4802] [Projects and Issues] [answered]
I think that the auto-summary for markdown files was indented based on the headings of the file.

It seems it is not the case anymore.

![image](https://user-images.githubusercontent.com/24386780/127118112-7dceaff6-a4e6-4780-9094-9a0ed05aecd4.png)

Is it on-purpose?

Auto-summaries with the indent was a really great replacement for embedded table of contents.
It is less obvious without the indent IMO

**Comment by darkred** [comment, marked as answer]
It's fixed now:
![2021-08-04_124826](https://user-images.githubusercontent.com/723651/128160482-3cb194a9-4c59-4caf-9b19-a40220b874ba.jpg)

**Reply by aurelien-reeves** [reply]
Indeed

Great, thanks!
